Cabal/Cabal.cabal

```diff
@@ -315,6 +315,7 @@ library
     Distribution.Compat.Exception
     Distribution.Compat.Graph
     Distribution.Compat.Internal.TempFile
+    Distribution.Compat.Prelude
```

Comment:
So, when the Distribution.Compat.Prelude was originally added @phadej specifically didn't export it because he didn't want any clients to depend on it. I'm pretty ambivalent about exposing it; I definitely don't want Custom setups depending on it unless we commit it as an API. But I understand why it makes your job here a bit easier. Not sure what to do!